docs/user-guide/systemrequirements.md

```diff
@@ -69,53 +68,58 @@ If you do not see your product listed here, please contact the Zowe community so
   - `ZWE_REFERRER_HOSTS=host1,host2,...`
-   
+
 See [Reviewing the instance env file](../user-guide/configure-instance-directory.md#reviewing-the-instanceenv-file) for additional information on the use of `instance.env` files.
 
-See [Configuring a Zowe instance via instance.env file](../user-guide/configure-instance-directory.md#configuring-a-Zowe-instance-via-instanceenv-file) for additional information on configuring `instance.env` files. 
+See [Configuring a Zowe instance via instance.env file](../user-guide/configure-instance-directory.md#configuring-a-Zowe-instance-via-instanceenv-file) for additional information on configuring `instance.env` files.
 
 ### Multi-Factor Authentication for Zowe Desktop
 
 To enable multi-factor authentication, you must install [IBM Z Multi-Factor Authentication](https://www.ibm.com/us-en/marketplace/ibm-multifactor-authentication-for-zos). For information on using MFA in Zowe, see [Multi-Factor Authentication](mvd-configuration.md#multi-factor-authentication-configuration).
 
 ### Using web tokens for SSO on ZLUX and ZSS
 
-In order to use web tokens for SSO on ZLUX and ZSS, users must first create a PKCS#11 token. See [Creating a PKCS#11 Token](configure-certificates.md#creating-a-pkcs11-token) for more information.   
+In order to use web tokens for SSO on ZLUX and ZSS, users must first create a PKCS#11 token. See [Creating a PKCS#11 Token](configure-certificates.md#creating-a-pkcs11-token) for more information.
 
-## Zowe CLI requirements (client)
+## Zowe CLI requirements
 
-### Client-side requirements
+### Client-side
 
-Zowe CLI is supported on platforms where Node.js 8.0 or higher is available, including Windows, Linux, and Mac operating systems. 
+Zowe CLI is supported on Windows, Linux, and Mac operating systems. Meet the following requirements before you install the CLI:
 
-- Install [Node.js V8.0 or higher LTS versions](https://nodejs.org/en/download/)
+- **Node.js:** Install a currently supported Node.js LTS version. For an up-to-date list of supported LTS versions, see [Node.js.org](https://nodejs.org/en/download/releases/).
 
-    **Tip:** You might need to restart the command prompt after installing Node.js. Issue the command `node --version` to verify that Node.js is installed.
+    ::: tip
+    You might need to restart the command prompt after installing Node.js. Issue the command `node --version` to verify that Node.js is installed.
+    :::
 
-- Install a version of **Node Package Manager (npm)** that is compatible with your version of Node.js. For a list of compatible versions, see [Node.js Previous Releases](https://nodejs.org/en/download/releases/).
+- **npm:** Ensure that you install a version of Node Package Manager (npm) that is compatible with your version of Node.js.
 
-    **Tip:** npm is included with the Node.js installation. Issue the command `npm --version` to verify the version of npm that is installed.
+    ::: tip
```

Comment:
This `::: tip` notation looks odd to me when previewing the Markdown. Will it look different on the docs site after being parsed by VuePress?

Reply:
Yes, just realized that I could do that and they generate like this: 
![image](https://user-images.githubusercontent.com/36675406/96748453-8b1f3600-1397-11eb-931f-65a994c2e701.png)